RecoEgamma/PhotonIdentification/plugins/PhotonMVAEstimatorRunIIFall17.cc

```diff
@@ -0,0 +1,312 @@
+#include "RecoEgamma/PhotonIdentification/plugins/PhotonMVAEstimatorRunIIFall17.h"
+#include "FWCore/ParameterSet/interface/FileInPath.h"
+#include "TMVA/MethodBDT.h"
+#include <vector>
+
+PhotonMVAEstimatorRunIIFall17::PhotonMVAEstimatorRunIIFall17(const edm::ParameterSet& conf):
+  AnyMVAEstimatorRun2Base(conf),
+  MethodName_("BDTG method"),
+  phoChargedIsolationLabel_(conf.getParameter<edm::InputTag>("phoChargedIsolation")), 
+  phoPhotonIsolationLabel_(conf.getParameter<edm::InputTag>("phoPhotonIsolation")), 
+  phoWorstChargedIsolationLabel_(conf.getParameter<edm::InputTag>("phoWorstChargedIsolation")), 
+  rhoLabel_(conf.getParameter<edm::InputTag>("rho")),
+  effectiveAreas_( (conf.getParameter<edm::FileInPath>("effAreasConfigFile")).fullPath()),
+  phoIsoPtScalingCoeff_(conf.getParameter<std::vector<double >>("phoIsoPtScalingCoeff"))
+  // phoIsoCutoff_(conf.getParameter<double>("phoIsoCutoff"))
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

Reply:
Changes implemented here:
https://github.com/lsoffi/cmssw/blob/CMSSW_9_4_0_pre3_TnP/RecoEgamma/PhotonIdentification/plugins/PhotonMVAEstimatorRunIIFall17.cc